### DISCUSSION TITLE
Worst update

### DISCUSSION THREAD

**Discussion by MadBreakerAI** [bmaltais/kohya_ss#1327] [General]
Worst update ever since last 2 months, already broke everything in lora making, now even worst for SDXL, thanks for your non-work! Cant fucking stand anymore this shitty tool not allowing me to train.

**Comment by 5nail000** [comment]
> Cant fucking stand anymore

XXXXDDDDD
- And since I can't stand it anymore, I'll have to throw a tantrum....

Сool strategy! You will change this world, we believe in you, only with you - it will become better

**Comment by TheEliteGeek** [comment]
OP: "I'm so entitled that when the **free** software I use breaks after an update, I'm allowed to insult the developers and not provide real feedback or solutions."

Bruh. This is a completely brand new technology and an incredibly small fraction of the entire Earth's population knows how to write code and develop code specifically for it.

If you want something that works, go learn computer science including all the math, AI, Python, and all sorts of other stuff and write your own damn SD training app.
